std/net/curl.d

```diff
@@ -3973,7 +3969,7 @@ struct Curl
 {
     alias OutData = void[];
     alias InData = ubyte[];
-    bool stopped;
```

Comment:
access is provided, see below. Though we might want to deprecate that.
Setting access should trigger an error as you shouldn't be able to set the internal state.